### PR TITLE
Add additional tests for cases where there is no markdown

### LINKED ISSUE #5: Initial validation
We should validate the input notebook:

- [ ] if there are no markdown cells, show an error
- [ ] if there are only H1 headings

### PR DESCRIPTION
**Pull request by 94rain**
## Describe your changes
Add additional tests for cases where there is no markdown

## Issue ticket number and link
Closes #5

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have added thorough tests (when necessary).
- [x] I have added the right documentation (when needed). Product update? If yes, write one line about this update.

### CODE REVIEW

File: tests/test_split.py [hunk 1]
```diff
@@ -117,8 +99,13 @@ def test_find_breaks_error_if_no_h2_but_h1_headers(tmp_empty):
     'only H2 headings are supported at this time.' in str(excinfo.value)
 
 
-def test_find_breaks_error_if_no_h1_and_h2_headers(tmp_empty):
-    nb = _read(no_h1_and_h2_headers)
+@pytest.mark.parametrize('md', [
```
Comment (idomic) [inline tests/test_split.py:102]: Nice, looks way better.
You can also add this test into it: test_find_breaks_warning_if_only_one_h2_header
Just make sure you parameterize the string you're looking in the output as well.

**Review by idomic — CHANGES_REQUESTED** [review]

File: tests/test_split.py (hunk 1 above)
Comment (94rain) [reply tests/test_split.py:102]: Do we want to merge `test_find_breaks_error_if_no_h2_but_h1_headers` `test_find_breaks_error_if_no_h1_and_h2_headers` `test_find_breaks_warning_if_only_one_h2_header` into one method, and put the warning texts as variables before methods?

Comment (idomic) [reply tests/test_split.py:102]: @94rain yes exactly! Any test case that repeats this pattern.

Comment (94rain) [reply tests/test_split.py]: I put them into two methods `test_find_breaks_errors` and `test_find_breaks_warnings` as they cannot be handled with the same logic.

**Comment by idomic** [comment]
lgtm, nice work!

**Pull request merged**